adaptive_navigation/lib/scaffold.dart

```diff
@@ -312,7 +330,8 @@ class AdaptiveNavigationScaffold extends StatelessWidget {
     return Row(
       children: [
         Drawer(
-          child: Column(
+          child: ListView(
```

Comment:
This might be OK but maybe should be a separate PR since it's not related to these issues

Reply:
Done